src/windshaft-integration/model-updater.js

```diff
@@ -164,6 +167,8 @@ ModelUpdater.prototype._updateLegendModel = function (legendModel, layerMetadata
 
 ModelUpdater.prototype._updateDataviewModels = function (windshaftMap, sourceId, forceFetch) {
   this._dataviewsCollection.each(function (dataviewModel) {
+    this._dataviewsTracker.add(dataviewModel);
```

Comment:
Instead of having to add this manually here... couldn't we pass the collection of dataviews to the `dataviewsTracker` so that it takes care of tracking dataviews?